CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+# CHANGELOG 1.9.0
+
+# v1.9.0 (2022-07-28)
+
+## OS Changes
+
+### General
+
+* SELinux policy now suppresses audit for tmpfs relabels ([#2222])
+* Restrict permissions for /boot and System.map ([#2223])
+* Remove unused crates growpart and servicedog ([#2238])
+* Rust dependency updates ([#2303])
+* Go dependency updates ([#2299])
+* Migrate to using `aws-sdk-rust` for first-party rust packages ([#2300])
```

Comment:
Since the build tools are still using `rusoto` I might say first-party OS rust packages (just to make it more clear).